jobserver/models/reports.py

```diff
@@ -0,0 +1,48 @@
+from django.db import models
+from django.urls import reverse
+from django.utils import timezone
+from django_extensions.db.fields import AutoSlugField
+
+
+class Report(models.Model):
+    release_file = models.ForeignKey(
+        "ReleaseFile",
+        on_delete=models.PROTECT,
+        related_name="reports",
+    )
+
+    title = models.TextField()
+    slug = AutoSlugField(max_length=100, populate_from=["title"], unique=True)
+    description = models.TextField()
+
+    authors = models.TextField(null=True)
```

Comment:
I think we should leave authors and doi out until we start using this for Reports Reports, as they will be easy to add later and we might end up implementing it differently potentially.

Reply:
Good point 👍